RevenueCatUI/Data/Strings.swift

```diff
@@ -16,6 +16,7 @@ import RevenueCat
 
 // swiftlint:disable identifier_name
 
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
```

Comment:
Oops thanks, not sure where I got this.

Reply:
Oh this is because some method I needed from URLSession wasn't available until specifically these versions. So having a lower watchOS one doesn't matter, it's just overly available. But I'll update for consistency.